### DISCUSSION TITLE
Installation fails

### DISCUSSION THREAD

**Discussion by MinyeZhou429** [aertslab/scenicplus#289] [Q&A]
Hi,

I'm trying to install scenicplus on google colab. It usually works well but today it gives me this error. I've updated pip but still doesn't work. My python version is 3.10.12.
<img width="503" alt="Screen Shot 2024-01-22 at 12 58 56 PM" src="https://github.com/aertslab/scenicplus/assets/112908002/b373d559-901e-4819-97cd-166617220608">

**Comment by SeppeDeWinter** [comment]
Hi @MinyeZhou429 

Please provide the full output of the error message.

I recently added a requirement for Rust to pycisTopic, so this might be the cause of the issue.

Please refer to this issue: https://github.com/aertslab/scatac_fragment_tools/issues/1

All the best,

Seppe

**Reply by MinyeZhou429** [reply]
Hi Seppe,

The screenshot is full error message. It doesn't show any specific information. I tried both on google colab and locally, both fail. It seems the issue is in scatac_fragment_tools because before the latest commit in scatac repo, everything works fine. I also checked my cmake is installed.

**Reply by SeppeDeWinter** [reply]
Hi @MinyeZhou429 

What is your Rust version?

Best,

Seppe
